### PR TITLE
Changed texts in queued package item that indecate current installing…

### PR DESCRIPTION
**Pull request by volodymyrhudyma**
… status

### CODE REVIEW

File: ota-plus-web/app/reactapp/src/components/device/queue/MtuListItem.jsx [hunk 1]
```diff
@@ -32,9 +32,9 @@ class MultiTargetITem extends Component {
                         {status === "waiting" ?
                             <span>Queued, waiting for device to connect</span>
                         : status === "downloading" ?
-                            <span>Downloading</span>
+                            <span>Waiting for Download to complete</span>
```
Comment (jerrytrieu) [inline ota-plus-web/app/reactapp/src/components/device/queue/MtuListItem.jsx]: No need to capitalize `Download`? @EpicNuts @christianhuggerATS

Comment (EpicNuts) [reply ota-plus-web/app/reactapp/src/components/device/queue/MtuListItem.jsx]: @jerrytrieu Sorry, that was my mistake. I should have passed it through Merlin, I guess?

Comment (volodymyrhudyma) [reply ota-plus-web/app/reactapp/src/components/device/queue/MtuListItem.jsx]: @jerrytrieu should I change that?

Comment (jerrytrieu) [reply ota-plus-web/app/reactapp/src/components/device/queue/MtuListItem.jsx]: Yes, and `Install` too below.

**Pull request merged**